src/rdf.js

```diff
@@ -43,3 +44,26 @@ export const n3FromJsonld = (jsonld, asTurtle) => {
 export const ttlFromJsonld = (jsonld) => {
   return n3FromJsonld(jsonld, true)
 }
+
+export const validateJsonld = async (req, resp, next) => {
```

Comment:
"validate" implies we are checking for "valid" jsonld rather than just parseable.  Maybe add a jsdoc comment?

Reply:
Agree. I'm going to rename.